lib/measures/tbd_monkey_patch/measure.rb

```diff
@@ -22,7 +22,7 @@
 
 require 'openstudio-standards'
 
-if OpenstudioStandards::VERSION >= '0.6.0' and OpenstudioStandards::VERSION <= '0.6.3'
+if OpenstudioStandards::VERSION >= '0.6.0'
```

Comment:
Works locally. All good. Thanks @macumber
Hadn't re-checked this, as it's working just fine for SDK v3.8.0.